docs: expand community docs: contributor - recommended tools (#1777)
asyncapi/community: recommended_tools.md

# Contributor - Recommended Tools
This article covers the recommended tools you should have as a contributor. With these tools, you can set up workflows and begin making contributions.

These are the recommended tools you need:-
- A laptop or a desktop computer capable of running tools which you would use for contribution.
- A stable internet connection to fork, clone a repo, make changes, and submit a PR.
- Install [Git](https://git-scm.com)- a version control system.
- [A GitHub account](https://github.com). AsyncAPI hosts all its project source code and dependencies on GitHub. You'll need a GitHub account to create issues, fork the repository, submit pull requests, and more. If you're new to GitHub, familiarize yourself with [basic GitHub functionalities and workflows](https://docs.github.com/en/get-started).
- A code editor, such as [VS Code](https://code.visualstudio.com).

> - [Nodejs and npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm). So you can run the website locally on your machine.

Reviewer: ```suggestion
- [Node.js and npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) to build and run the website locally.
```

> - A working browser such as Google Chrome, Firefox or Microsoft Edge where you can preview your changes. 

Reviewer: ```suggestion
- A web browser, such as Google Chrome, Firefox or Microsoft Edge to preview your changes. 
```

> - A working browser such as Google Chrome, Firefox or Microsoft Edge where you can preview your changes. 

Reviewer: What about previewing Markdown documents that do not require spinning up a website and/or opening a browser?

> - Lastly, you need a command line terminal where you can run commands. You can use your local command line or the terminal in VSCode.

Reviewer: I would place it right after the code editor and before Node.js

```suggestion
- Command-line terminal, such as the one included in your OS, or the terminal integrated in your code editor or IDE, to run commands.
```
